float32 resolutionMap
float32 resPoseIndoor
float32 resPoseOutdoor
float32 overlapThre
---
bool success
